### PR TITLE
Views

### PR DESCRIPTION
**Pull request by quanghv96**

### CODE REVIEW

**Comment by quanghv96** [comment]
ready

**Comment by quanghv96** [comment]
ready

**Comment by quanghv96** [comment]
ready

**Comment by quanghv96** [comment]
ready

File: database/seeds/UsersTableSeeder.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+    	DB::transaction(function() {
+			factory(App\Address::class, 1)->create([
```
Comment (BunnyPi04) [inline database/seeds/UsersTableSeeder.php]: tab lại

**Comment by BunnyPi04** [comment]
commented

**Pull request closed without merging**